.drone.jsonnet

```diff
@@ -1,18 +1,20 @@
-local Pipeline(os, arch, version) = {
+local Pipeline(os, arch, version, alpine=false) = {
     kind: "pipeline",
-    name: os+" - "+arch+" - Julia "+version,
+    name: os+" - "+arch+" - Julia "+version+(if alpine then " (Alpine)" else ""),
     platform: {
-	os: os,
-	arch: arch
+        os: os,
+        arch: arch
     },
     steps: [
-	{
-	    name: "build",
-	    image: "julia:"+version,
-	    commands: [
-		"julia --project=. --check-bounds=yes --color=yes -e 'using InteractiveUtils; versioninfo(verbose=true); using Pkg; Pkg.test(coverage=true)'"
-	    ]
-	}
+        {
+            name: "Run tests",
+            image: "julia:"+version+(if alpine then "-alpine" else ""),
+            commands: [
+                "julia --project=. --check-bounds=yes --color=yes -e 'using InteractiveUtils; versioninfo(verbose=true)'",
+                "julia --project=. --check-bounds=yes --color=yes -e 'using Pkg; Pkg.build()'",
```

Comment:
We don't need to call `Pkg.build()` anymore

Reply:
Uh, right, I used my standard template for CI of julia packages on Drone with all the steps.  I removed (again) this one, thanks!